spec_template.md

```diff
@@ -1,31 +1,80 @@
 > See comments in Markdown for tips on how to use this spec template
+### YOUR FEATURE TITLE
+<!-- Update with a concise title describing your feature. -->
+
+### Feature Pitch Link
+https://github.com/Microsoft/calculator/Issues/<issueNum>
+<!--
+Update this to reference the approved feature pitch, as that is where ongoing discussion will occur.
+-->
 
 ### Problem Statement
-<!-- What problem are we trying to solve? Who’s the target audience? Is there a customer need or pain point we need to remedy? Is there a business goal or metric we are trying to improve? Do we have a hypothesis we want to prove or disprove? -->
+<!--
+What problem are we trying to solve? Who’s the target audience? Is there a customer need or pain
+point we need to remedy? Is there a business goal or metric we are trying to improve? Do we have a
+hypothesis we want to prove or disprove?
+-->
 
 ### Evidence or User Insights
-<!-- Why should we do this? Potential sources of data: Feedback Hub, other GitHub issues, other anecdotes from listening to customers in person or online, request from another team, telemetry data, user research, market or competitive research -->
+<!--
+Why should we do this? Potential sources of data: Feedback Hub, other GitHub issues, other anecdotes
+from listening to customers in person or online, request from another team, telemetry data,
+user research, market or competitive research.
```

Comment:
Do not manually wrap comments.

Reply:
On this one I disagree.  Users will be downloading this file and using it as the basis of their spec.  We should prevent the need for horizontal scrolling when they are editing their content.